src/EnergyPlus/HVACCooledBeam.cc

```diff
@@ -678,7 +661,6 @@ namespace HVACCooledBeam {
         static constexpr std::string_view RoutineName("SizeCoolBeam");
         int PltSizCoolNum(0);          // index of plant sizing object for the cooling loop
         int NumBeams(0);               // number of beams in the zone
-        int Iter(0);                   // beam length iteration index
```

Comment:
[src/EnergyPlus/HVACCooledBeam.cc:681]:(style),[variableScope],The scope of the variable 'Iter' can be reduced.